test/fixtures/gateway_account_fixtures.js

```diff
@@ -45,6 +45,22 @@ function validGatewayAccount (opts) {
 }
 
 module.exports = {
+  validGatewayAccountPatchRequest: (opts = {}) => {
```

Comment:
I would be inclined to just have the arguments as `(path, value)` rather than wrapped in an object `opts` object but I don't want to hold you up.